hazelcast-client/src/main/java/com/hazelcast/client/connection/nio/ClusterConnectorServiceImpl.java

```diff
@@ -374,6 +374,7 @@ public void run() {
 
                     setOwnerConnectionAddress(null);
                     connectionStrategy.onDisconnectFromCluster();
+                    client.onClusterDisconnect();
```

Comment:
previous line uses method naming as `onDisconnectFromCluster` and yet the new introduced method at this line is named `onDisconnectFromCluster `. I just wish that they are consistent and the same naming.

Reply:
This is meant to be consistent with `client.onClusterConnect`